front/lib/models/data_source.ts

```diff
@@ -78,6 +79,10 @@ DataSource.init(
       type: DataTypes.STRING,
       allowNull: false,
     },
+    dustAPIDataSourceId: {
```

Comment:
FYI, as I read it here, dustAPI was confusing to me (backend endpoints are also "dust" api and public api is also "dust" api).

I see it's coherent with dustAPIProjectId but why not coreApiDataSourceId intially ? (we use CoreApi for calling it).

Reply:
Yes yes this naming is confusing but I favor consistency here :+1: 

Should be coreProjectId and coreDataSourceId :+1: Can be tackled orthogonally of course.